packages/reducers/src/makeEnhancer.test.js

```diff
@@ -1,11 +1,13 @@
 import { identity } from 'ramda';
 import { FUNCTION_KEY } from '@redux-tools/injectors';
 import { DEFAULT_FEATURE } from '@redux-tools/namespaces';
+import { createStore as createStoreRedux } from 'redux';
 
 import makeEnhancer, { storeInterface } from './makeEnhancer';
 
 const createStore = () => ({
 	replaceReducer: jest.fn(),
+	state: {},
```

Comment:
i dont think this line is necessary

besides, a store doesnt have state, but a getState method